docs/users/input_ui.md

```diff
@@ -36,10 +35,7 @@ If a line layer is selected, a path will be generated from the GPS track
 until the record button is pressed again. The form for the recorded line
 will appear accordingly.
 
-Input sets the last surveyed layer as the default layer for editing.
-Subsequent pressing of the record will make changes to the default
-layer. To reset the default layer and see the full list of the survey
-layers, you need to press and hold the record button.
+Pressing and holding ** Record ** button will allow you to set the default survey layer. This option is also available under the **Settings** section.
```

Comment:
If you use spaces between text and asterisks, the text will not get bold ... use `**Record**` instead of `** Record **`